### ISSUE TITLE
User can't specify to which tag he wants to add a uncertainty when `locus=name` in Annotator

### ISSUE THREAD

**Issue by bug-rancher** [providedh/collaborative-platform#81]
When user wants to add uncertainty to a tag (case 4 from table: https://docs.google.com/spreadsheets/d/1zCEhS2YHczPMYaob_IKWx-s7vz1qULEhcND51wP6CXk), they should specify name of the tag to append uncertainty and optionally propose better value in `Asserted value` field. Request sent to backend should have fields:
```
{
  "start_pos": int,
  "end_pos": int,
  "locus": "name",
  "tag": str,  # name of the tag to append uncertainty
  "asserted_value": str,  # optional, if user have a proposition of better tag, he put it here
  "categories": [str, ...],
  "certainty": str,
  "desc": str  # optional
}
```
For now there is no possibility to define to which tag user wants to append uncertainty and put a optional proposition to `asserted_value`, because there is no field `Tag name` in interface (when `Locus` is set to `name`), and value from `Asserted value` is send in request in `tag` field.

**Comment by Janchorizo** [comment]
According to the [Annotating TEI with uncertainty](https://docs.google.com/document/d/16jWs-AQ7XXpUUnH6lvH2Hjv8bhezi23lLC7FOM1IbNo/edit#) document, the following message does work.
```
{
"start_row": 217,
"start_col": 73,
"end_row": 217,
"end_col": 92,
"category": "incompleteness",
"locus": "name",
"certainty": "high",
"asserted_value": "",
"description": "",
"tag": "surname"
}
```
This is the current implementation. I will change it to provide the tag name, although there is no
use case where it would be needed (and making the user specifying it is confusing). Nested tags
do not attend to the use case of annotating plain text, nor we have tags that should be nested inside 
each other.

I propose making use of the Scenario 5.2.1.1 mentioned in the file, where I send the tag name
as the asserted value. In case that there is already a tag completely surrounding the entity, such 
would get the annotation.
the annotation as saaid

**Comment by bug-rancher** [comment]
>According to the Annotating TEI with uncertainty document, the following message does work.

This document is out of date. We simplified all this scenarions to 6 cases presented in table: https://docs.google.com/spreadsheets/d/1zCEhS2YHczPMYaob_IKWx-s7vz1qULEhcND51wP6CXk . That will be simpler for the user and Annotator logic is now simpler too.

>This is the current implementation. I will change it to provide the tag name, although there is no use case where it would be needed (and making the user specifying it is confusing).

This is not confusing for the user. User just needs to know to which tag he wants to append uncertanty. This applies not only to this use case, but to every use case.

>Nested tags do not attend to the use case of annotating plain text, nor we have tags that should be nested inside each other.

But we need to predict the situation, that users adds a few tags to the same fragment of text, and then someone will want to add uncertainty to one of those. I think it's very probable.

In short, please don't follow the file https://docs.google.com/document/d/16jWs-AQ7XXpUUnH6lvH2Hjv8bhezi23lLC7FOM1IbNo , but this table instead: https://docs.google.com/spreadsheets/d/1zCEhS2YHczPMYaob_IKWx-s7vz1qULEhcND51wP6CXk . I think that will simplify frontend logic too.

**Comment by Janchorizo** [comment]
At least for the following workshops, I believe adding this input is not a good idea as it would
require giving further explanations for the simplest task of uncertainty annotation: locus = name.

I propose, if it suffices, that I send the selected tag name as the `asserted_value` parameter and the
`tag_name` parameter based on the annotation use case:
- If the user is annotating plain text, I will send the same tag name as the asserted value.
- I the user is annotating using a reference, the I will send the tag name of the referenced entity.
- If the user is annotating text inside an entity tag, then I will send the tag name of the surrounding tag.

**Comment by michalkozak** [comment]
Because we don't have to hurry with this task for the following workshop, I would connect this task with #35 and #86. 

I think we always should display "Tag name" combo box with entities chosen from the user (for locus=value and locus=name) but "Attribute name" input only for locus=value.
 
And when we have the properties of an entity, "Attribute name" could be combo box with the properties of chosen entity (in "Tag name" combo box) + sameAs + ref.

But let as think and write down all cases with the proposition from #35 to be sure that we cover all scenarios. @bug-rancher will write all scenarios because we will need additional sub-form for properties of entities.
